### PR TITLE
Add instructions for getting started with JSLint

### PR DESCRIPTION
**Pull request by rugk**
https://github.com/AtomLinter/linter-eslint/issues/378#issuecomment-166166366

[Review on Reviewable](https://reviewable.io/reviews/atomlinter/linter-eslint/381)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,15 @@ You have two options:
   * Activate `Use Global Eslint` package option
   * (Optional) Set `Global Node Path` with `$ npm config get prefix`
 
+## Using JSLint
```
Comment (steelbrain) [inline README.md]: It's not `JSLint`, it's `ESLint`

File: README.md [hunk 2]
```diff
@@ -36,6 +36,15 @@ You have two options:
...
+
+Note that recent versions of JSLint do not use any rules by-default. This means you have to specify a configuration file for your project!
+To do this in a straightforward way run this:
+```ShellSession
+eslint --init
+```
+
+Alternatively you can create the `.eslintrc` file by yourself. In any way it is a good idea to have a look at the [Get Started With ESLint](http://devnull.guru/get-started-with-eslint/) blog post by [IanVS](https://github.com/IanVS) and obviously [the ESLint documentation](http://eslint.org/docs/user-guide/configuring), especially the [list of rules](http://eslint.org/docs/rules/).
```
Comment (steelbrain) [inline README.md]: `s/In any way/Anyways/`

**Comment by rugk** [comment]
Review status: 0 of 1 files reviewed at latest revision, 2 unresolved discussions.

---

*[README.md, line 39 \[r1\]](https://reviewable.io:443/reviews/atomlinter/linter-eslint/381#-K63IoGQSv8p_OVPo9X_-r1-39:-K63JHu9xubRzca1B1ej:1303385818) ([raw file](https://github.com/atomlinter/linter-eslint/blob/acfb2c0eb7f1d428f8680ca4b54a553c25983d32/README.md#L39)):*
Opps, you're right.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/atomlinter/linter-eslint/381)*

File: README.md [hunk 3]
```diff
@@ -36,6 +36,15 @@ You have two options:
...
 
+## Using ESLint
+
+Note that recent versions of ESLint do not use any rules by-default. This means you have to specify a configuration file for your project!
+To do this in a straightforward way run this:
+```ShellSession
+eslint --init
+```
+
+Alternatively you can create the `.eslintrc` file by yourself. Anyway it is a good idea to have a look at the [Get Started With ESLint](http://devnull.guru/get-started-with-eslint/) blog post by [IanVS](https://github.com/IanVS) and obviously [the ESLint documentation](http://eslint.org/docs/user-guide/configuring), especially the [list of rules](http://eslint.org/docs/rules/).
```
Comment (steelbrain) [inline README.md]: Can you please remove the words `obviously` and `especially from here?`

**Comment by rugk** [comment]
Review status: 0 of 1 files reviewed at latest revision, 3 unresolved discussions.

---

*[README.md, line 39 \[r1\]](https://reviewable.io:443/reviews/atomlinter/linter-eslint/381#-K63IoGQSv8p_OVPo9X_-r1-39:-K63JlGSQKPv4B7r0mAk:-499146009) ([raw file](https://github.com/atomlinter/linter-eslint/blob/acfb2c0eb7f1d428f8680ca4b54a553c25983d32/README.md#L39)):*
Done.

---

*[README.md, line 47 \[r1\]](https://reviewable.io:443/reviews/atomlinter/linter-eslint/381#-K63IoGQSv8p_OVPo9X_-r1-47:-K63Jio15Hg3yR1xdFmv:-499146009) ([raw file](https://github.com/atomlinter/linter-eslint/blob/acfb2c0eb7f1d428f8680ca4b54a553c25983d32/README.md#L47)):*
Done.

---

*[README.md, line 47 \[r2\]](https://reviewable.io:443/reviews/atomlinter/linter-eslint/381#-K63IoGQSv8p_OVPo9X_-r2-47:-K63JslguQudZJ--InEK:-1649340426) ([raw file](https://github.com/atomlinter/linter-eslint/blob/72daafcd87bb2b21d2cd9ab0451cf8543f7cdccb/README.md#L47)):*
Done. Better?

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/atomlinter/linter-eslint/381)*

File: README.md [hunk 4]
```diff
@@ -36,6 +36,15 @@ You have two options:
...
 
+## Using ESLint
+
+Note that recent versions of ESLint do not use any rules by-default. This means you have to specify a configuration file for your project!
+To do this in a straightforward way run this:
+```ShellSession
+eslint --init
+```
+
+Alternatively you can create the `.eslintrc` file by yourself. Anyway it is a good idea to have a look at the [Get Started With ESLint](http://devnull.guru/get-started-with-eslint/) blog post by [IanVS](https://github.com/IanVS) and [the ESLint documentation](http://eslint.org/docs/user-guide/configuring), including the [list of rules](http://eslint.org/docs/rules/).
```
Comment (steelbrain) [inline README.md]: One final change, can we drop this word `Anyway` completely?

**Comment by rugk** [comment]
Review status: 0 of 1 files reviewed at latest revision, 4 unresolved discussions.

---

*[README.md, line 47 \[r3\]](https://reviewable.io:443/reviews/atomlinter/linter-eslint/381#-K63IoGQSv8p_OVPo9X_-r3-47:-K63K6o-mzwDxVIcBdzs:1898978842) ([raw file](https://github.com/atomlinter/linter-eslint/blob/773dab3cb49a4a781f14111ecb9b4880b474b927/README.md#L47)):*
God idea. Done.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/atomlinter/linter-eslint/381)*

**Comment by rugk** [comment]
Review status: 0 of 1 files reviewed at latest revision, 4 unresolved discussions.

---

*[README.md, line 47 \[r3\]](https://reviewable.io:443/reviews/atomlinter/linter-eslint/381#-K63IoGQSv8p_OVPo9X_-r3-47:-K63KB1F6AhWG1qNNoaE:-594291982) ([raw file](https://github.com/atomlinter/linter-eslint/blob/773dab3cb49a4a781f14111ecb9b4880b474b927/README.md#L47)):*
*Good :smile: 

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/atomlinter/linter-eslint/381)*

**Comment by steelbrain** [comment]
LGTM, Thanks! Waiting on CI

**Pull request merged**